frigate/config.py

```diff
@@ -365,6 +368,9 @@ class BirdseyeConfig(FrigateBaseModel):
 # uses BaseModel because some global attributes are not available at the camera level
 class BirdseyeCameraConfig(BaseModel):
     enabled: bool = Field(default=True, title="Enable birdseye view for camera.")
+    position: int = Field(
```

Comment:
Should be renamed as `order` to match the existing `ui -> order` option

Reply:
Sure 👍